satellite/contact/endpoint.go

```diff
@@ -49,23 +47,32 @@ func (endpoint *Endpoint) CheckIn(ctx context.Context, req *pb.CheckInRequest) (
 
 	peerID, err := identity.PeerIdentityFromContext(ctx)
 	if err != nil {
+		endpoint.log.Info("failed to get node ID from context", zap.String("node address", req.Address), zap.String("error", err.Error()))
```

Comment:
Would it be ok to use `zap.Error(err)` instead of `zap.String("error", err.Error())`? That will avoid actually calculating the error message when it's not needed, and is more compact in code, but more importantly our logging may treat errors specially when they are marked as error fields.

Reply:
this is great, let me update that.